web/server/api/endpoints/environments.py

```diff
@@ -31,4 +30,32 @@ async def get_environments(
             start_at=c.EPOCH,
             plan_id="",
         )
-    return environments
+    if context.config.default_target_environment not in environments:
+        environments[context.config.default_target_environment] = Environment(
+            name=context.config.default_target_environment,
+            snapshots=[],
+            start_at=c.EPOCH,
+            plan_id="",
+        )
+    return ApiEnvironmentsPayload(
+        environments=environments,
+        pinned_environments=context.config.pinned_environments,
+        default_target_environment=context.config.default_target_environment,
+    )
+
+
+@router.delete("/{environment:str}")
+async def delete_environment(
+    context: Context = Depends(get_loaded_context),
+    environment: str = "",
```

Comment:
Should this not have a default value so it's required? What happens if an empty string is passed to invalidate_environment()?